examples/istio/README.md

```diff
@@ -0,0 +1,83 @@
+# A TiDB cluster with monitoring in an Istio enabled kubernetes cluster
+
+> **Note:**
+>
+> This setup is for test or demo purpose only and **IS NOT** applicable for critical environment. Refer to the [Documents](https://pingcap.com/docs/stable/tidb-in-kubernetes/deploy/prerequisites/) for production setup.
+
+The following steps will create a TiDB cluster with monitoring, the monitoring data is not persisted by default.
+
+**Prerequisites**: 
+- Has TiDB operator `v1.1.0-beta.1` or higher version installed. [Doc](https://pingcap.com/docs/stable/tidb-in-kubernetes/deploy/tidb-operator/)
+- Has default `StorageClass` configured, and there are enough PVs (by default, 6 PVs are required) of that storageClass:
+  
+  This could by verified by the following command:
+  
+  ```bash
+  > kubectl get storageclass
+  ```
+  
+  The output is similar to this:
+  
+  ```bash
+  NAME                 PROVISIONER               AGE
+  standard (default)   kubernetes.io/gce-pd      1d
+  gold                 kubernetes.io/gce-pd      1d
+  ```
+  
+  Alternatively, you could specify the storageClass explicitly by modifying `tidb-cluster.yaml`.
+
+## Disable MTLS
+
+```bash
+> kubectl -n <namespace> apply -f disable-mtls.yaml
```

Comment:
How about adding a comment to replace `<namespace>` in `disable-mtls.yaml`?

Reply:
@csuzhangxc Done